src/corebluetooth.cs

```diff
@@ -46,9 +46,23 @@ interface CBManager {
 		[Export ("state", ArgumentSemantic.Assign)]
 		CBManagerState State { get; }
 
-		[iOS (13,0), TV (13,0), Watch (6,0), Mac (10, 15)]
+#if IOS
+		[Internal]
+		[iOS (13,0)]
+		[Export ("authorization", ArgumentSemantic.Assign)]
+		CBManagerAuthorization _IAuthorization { get; }
+
+		[Internal]
+		[iOS (13,1)]
+		[Static]
+		[Export ("authorization", ArgumentSemantic.Assign)]
+		CBManagerAuthorization _SAuthorization { get; }
+#else
```

Comment:
Noob question 😅Does `[iOS (13,0)]` not mean anything 13+? Then are there two Exports for the same selector? How does it get resolved?

Reply:
@prmallic good questions :)

Yes, `[iOS]` is a shortcut for `[Introduced (PlatformName.iOS, 13, 0)]` which means it's the first version where the API is available.

The _exported_ strings are identical but the selector is not. Static selector are prefixed with `+` so they can't conflict with the instance selectors of the same name. Our `[Export]` attribute hides that bit (it's implied with the `[Static]` attribute).